infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration scan="true" scanPeriod="30 seconds">
+
+    <property name="LOG_DIRECTORY" value="/var/log/corfu" />
+
+    <appender name="file" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${LOG_DIRECTORY}/corfu.9000.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+            <fileNamePattern>${LOG_DIRECTORY}/corfu.9000.%i.log.zip</fileNamePattern>
```

Comment:
You can use `.tgz` to be consistent with the rest of the logs.

Reply:
Replaced zip by tar.gz
done